Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -193,6 +193,38 @@ Command keywords and values
          make a copy of the record on the same page.
 
 
+ - :Command:
+         copyToLanguage
+   :Type:
+         integer
+   :Value:
+         A new DataHandler command 'copyToLanguage' has been introduced. It behaves like 'localize' command
+         (both record and child records are copied to given language), but does not set transOrigPointerField fields (e.g. l10n_parent).
```

Comment:
Please add php syntax highlighting to `transOrigPointerField`.

Reply:
ok